inst/templates/ad_adlb.R

```diff
@@ -0,0 +1,210 @@
+# Name: ADLB
+#
+# Label: Laboratory Analysis dataset for metabolic trials
+#
+# Input: adsl_metabolic, lb_metabolic, admiralmetabolic_advs
+
+# Attach/load required packages ----
+library(admiral)
+library(admiralmetabolic)
+library(pharmaversesdtm)
+library(dplyr)
+library(stringr)
+
+# Define project options/variables ----
+# Use the admiral option functionality to store subject key variables in one
+# place (note: `subject_keys` defaults to STUDYID and USUBJID)
+set_admiral_options(subject_keys = exprs(STUDYID, USUBJID))
+
+# Store ADSL variables required for derivations as an R object, enabling
+# simplified usage throughout the program
+adsl_vars <- exprs(TRTSDT, TRTEDT, TRT01A, TRT01P)
+
+# Read in and prepare data ----
+# See the "Read in Data" vignette section for more information:
+# (https://pharmaverse.github.io/admiral/articles/bds_finding.html#readdata)
+lb_metabolic <- pharmaversesdtm::lb_metabolic
+admiralmetabolic_adsl <- admiralmetabolic::admiralmetabolic_adsl
+admiralmetabolic_advs <- admiralmetabolic::admiralmetabolic_advs
+
+# Convert SAS missing character values to NA
+adlb <- lb_metabolic %>%
+  convert_blanks_to_na()
+adsl <- admiralmetabolic_adsl %>%
+  convert_blanks_to_na()
+advs <- admiralmetabolic_advs %>%
+  convert_blanks_to_na()
+
+# Remove glucose and insulin measurements collected in an un-fasted state
+adlb <- adlb %>%
+  filter(!(LBTESTCD %in% c("GLUC", "INSULIN") & LBFAST != "Y"))
+
+# Merge ADSL variables (stored in `adsl_vars`) needed for ADLB
+adlb <- adlb %>%
+  derive_vars_merged(
+    dataset_add = adsl,
+    new_vars = adsl_vars,
+    by_vars = get_admiral_option("subject_keys")
+  )
+
+# Add parameter information ----
+# Define parameter look-up table used for merging parameter codes to ADLB
+param_lookup <- tribble(
```

Comment:
```suggestion
param_lookup <- tibble::tribble(
```
or add {tibble} in section # Attach/load required packages

Reply:
Ill add to top :)